app/javascript/mastodon/components/server_banner.js

```diff
@@ -62,23 +62,25 @@ class ServerBanner extends React.PureComponent {
             <Account id={server.getIn(['contact', 'account', 'id'])} />
           </div>
 
-          <div className='server-banner__meta__column'>
-            <h4><FormattedMessage id='server_banner.server_stats' defaultMessage='Server stats:' /></h4>
-
-            {isLoading ? (
-              <>
-                <strong className='server-banner__number'><Skeleton width='10ch' /></strong>
-                <br />
-                <span className='server-banner__number-label'><Skeleton width='5ch' /></span>
-              </>
-            ) : (
-              <>
-                <strong className='server-banner__number'><ShortNumber value={server.getIn(['usage', 'users', 'active_month'])} /></strong>
-                <br />
-                <span className='server-banner__number-label' title={intl.formatMessage(messages.aboutActiveUsers)}><FormattedMessage id='server_banner.active_users' defaultMessage='active users' /></span>
-              </>
-            )}
-          </div>
+          {activityApiEnabled && (
```

Comment:
This number is not connected to this API. Why this change?

Reply:
Oops, I misunderstood as I was displaying statistics. I will revert this change.